app/routers/event.py

```diff
@@ -148,3 +148,33 @@ def delete_event(request: Request,
         # if the deletion is successful
     return RedirectResponse(
         url="/calendar", status_code=status.HTTP_200_OK)
+
+
+def check_date_validation(start_time, end_time) -> bool:
+    """Check if the start_date is smaller then the end_time"""
+
+    try:
+        return start_time < end_time
+    except TypeError:
+        return False
+
+
+def add_new_event(values: dict, db) -> Optional[Event]:
+    """Get User values and the DB Session insert the values
+    to the DB and refresh it exception in case that the keys
+    in the dict is not match to the fields in the DB
+    return the Event Class item"""
+
+    if not check_date_validation(values['start'], values['end']):
+        return None
+    try:
+        new_event = create_model(
+                    db, Event, **values)
+        create_model(
+                db, UserEvent,
+                user_id=values['owner_id'],
+                event_id=new_event.id
+            )
+        return new_event
+    except (AssertionError, AttributeError, TypeError):
+        return None
```

Comment:
Please log something. It's considered a bad practice to silence errors :)

Reply:
I add logger.critical